docs/source/technical_tutorials/working_with_r/cmr_search_in_r.ipynb

```diff
@@ -15,7 +15,7 @@
    "source": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.        req_url_query("q" = paste("atlas", "vegetation height"), "hint_lang" = "python") |&gt;
</pre><p>you can remove the <code>hint_lang</code> parameter - I updated the API to always return code hints for Python and R. I probably need to update the federated search notebook with this change, too.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/495/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>